style.css

```diff
@@ -152,4 +152,46 @@ body{
     height: 300px;
     display: flex;
     flex-direction: column;
-}
\ No newline at end of file
+}
+
+.social-links{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    padding-top: 15px;
+}
+
+.insta-link{
+    padding: 5px;
```

Comment:
Also primo use `em` and not px 

This is better for different screen resolutions and sizes